update-db.js

```diff
@@ -65,6 +65,41 @@ function getLatestInfo (lock) {
   }
 }
 
+function getBrowsersList () {
+  return childProcess.execSync('npx browserslist').toString()
+    .trim()
+    .split('\n')
+    .map(function (line) { return line.trim().split(' ') })
+    .reduce(function (result, entry) {
+      if (!result[entry[0]]) {
+        result[entry[0]] = []
+      }
+      result[entry[0]].push(entry[1])
+      return result
+    }, {})
+}
+
+function diffBrowsersLists (old, current) {
+  return Object.keys(old)
+    .sort()
+    .flatMap(function (browser) {
+      var intersection = old[browser].filter(function (version) {
```

Comment:
I'm sure that more sophisticated algorithm can be used, but taking into account number of version to compare it will be overkill

Reply:
Note, that there are versions with multiple numbers `10-11` (see `ios` versions)

Reply:
I would prefer the old solution, it's more readable and prefers pretty order. If we will find out that this places is really slow, then we can optimize it.